### PR TITLE
Document initializing PackedVectorArray/PackedColorArray correctly

### LINKED ISSUE #62927: PoolVector2Array: cannot initialize with array
### Godot version

3.4.4

### System information

Manjaro Linux, Vulkan, AMD RADV NAVI14

### Issue description

documentation claims that PoolVector2Array has optional way to initialize with generic array.
please see constructor description https://docs.godotengine.org/en/stable/classes/class_poolvector2array.html?highlight=poolvector2array
I've tried several ways to initialize and draw a Line2D but nothing worked except appending Vector2s
here is what I tried (commented out are the variants which didn't work):
```
shape = [[0, 0], [100, 0], [100, 100], [0, 100]]
func draw_shape(shape):
	var l = Line2D.new()
	l.width = 10
	l.points = make_pool_vector(shape)

#	l.points = PoolVector2Array(shape)
#      l.points = PoolVector2Array([[0, 0], [100, 0], [100, 100], [0, 100]])
#	l.points = PoolVector2Array([0, 0, 100, 0, 100, 100, 0, 100])

	add_child(l)

func make_pool_vector(data) -> PoolVector2Array:
	var pv: PoolVector2Array
	for i in data:
		pv.append(Vector2(i[0],i[1]))
	return pv
```

### Steps to reproduce

in the description

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by Calinou**
This closes https://github.com/godotengine/godot/issues/62927.

This can be remade for `3.x` once we agree on the text to add.

### CODE REVIEW

File: doc/classes/PackedVector2Array.xml [hunk 1]
```diff
@@ -5,6 +5,10 @@
 	</brief_description>
 	<description>
 		An array specifically designed to hold [Vector2]. Packs data tightly, so it saves memory for large array sizes.
+		[b]Note:[/b] When initializing a [PackedVector2Array] with elements, it must be initialized with an [Array] of [Color] values, not separate [float]s:
```
Comment (fire-forge) [inline doc/classes/PackedVector2Array.xml:8]:
```suggestion
		[b]Note:[/b] When initializing a [PackedVector2Array] with elements, it must be initialized with an [Array] of [Vector2] values, not separate [float]s:
```

I assume this is a copy and paste mistake?

File: doc/classes/PackedVector3Array.xml [hunk 2]
```diff
@@ -5,6 +5,10 @@
 	</brief_description>
 	<description>
 		An array specifically designed to hold [Vector3]. Packs data tightly, so it saves memory for large array sizes.
+		[b]Note:[/b] When initializing a [PackedVector3Array] with elements, it must be initialized with an [Array] of [Color] values, not separate [float]s:
```
Comment (fire-forge) [inline doc/classes/PackedVector3Array.xml:8]:
```suggestion
		[b]Note:[/b] When initializing a [PackedVector3Array] with elements, it must be initialized with an [Array] of [Vector3] values, not separate [float]s:
```

**Review by fire-forge — CHANGES_REQUESTED** [review]

**Review by fire-forge — APPROVED** [review]

**Comment by kleonc** [comment]
I think it would make more sense to add these notes to the descriptions of `Packed<Type>Array(Array from)` constructors instead. These notes are about just these specific constructors.

**Review by reduz — DISMISSED** [review]

**Review by akien-mga — COMMENTED** [review]
I think these should go in the description of the relevant constructor instead.
Not sure about mentioning "not separate floats", I know that's what the user tried to do in the relevant issue but that's not something that I would naturally expect to work, so I don't think we need to specifically point out this arbitrary possibility.

**Comment by Calinou** [comment]
I've amended the PR to put the descriptions in the specific constructors.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**